apps/functions/applications-background-jobs/notify-subscribers/src/notify-subscribers.js

```diff
@@ -75,12 +75,11 @@ export class NotifySubscribers {
 		if (!this.messageIsValid()) {
 			return;
 		}
-		const update = await this.getExtendedUpdate();
```

Comment:
the extended update is required to get the project name to be passed to the rest of the update data. Why was that undone?

Reply:
My fault, that should have stayed in.  I have restored those changes.